src/main/frontend/db_worker.cljs

```diff
@@ -110,7 +102,9 @@
                     #js {:$addr addr
                          :$content (pr-str data)})
                   addr+data-seq)]
-        (upsert-addr-content! repo data delete-addrs)))
+        ;; async write so that UI can be refreshed earlier
+        (async/go
+          (upsert-addr-content! repo data delete-addrs))))
```

Comment:
Writes to SQLite are async so that UI can be refreshed as fast as it can, will this be an issue for data-loss in some situations? @andelf @RCmerci